docs/koku_setup_on_m1.rst

```diff
@@ -7,30 +7,14 @@ About
 
 This documentation is to guide M1 Mac users on how to successfully install and run Koku on their machines.
 
-Generally, in order to run Intel-based packages and programs on Mac with Apple silicon processors (M1), one can make `use of Rosetta`_ which simulates the Intel environment.
+Generally, in order to run Intel-based packages and programs on Mac with Apple silicon processors (M1), one can make `use of Rosetta`_ which emulates the Intel environment.
```

Comment:
Should we just remove this file entirely?

Reply:
Oh, I thought I did that. Will do.

Reply:
We should probably also remove `rosetta_on_m1.rst`. That suggests some things that can get you in trouble, namely installing Python with `brew`. I don't think having x86 binaries is really necessary.